### PR TITLE
Adjust PLATFORM_NAME setting for Android app compatibility

### PR DESCRIPTION
**Pull request by Abdul-Muqadim-Arbisoft**

### CODE REVIEW

**Comment by DawoudSheraz** [comment]
This should be merged in either nightly or master.

**Comment by Abdul-Muqadim-Arbisoft** [comment]
> This should be merged in either nightly or master.

rebased to master

File: tutorandroid/templates/android/build/config/tutor.yaml [hunk 1]
```diff
@@ -68,7 +68,10 @@ BRANCH:
   ALTERNATE_HOST: ''
 
 #Platform names
-PLATFORM_NAME: "{{ PLATFORM_NAME }}"
+# TODO: Temporarily set PLATFORM_NAME based on OPENEDX_COMMON_VERSION to 'OpenEdx' for compatibility with the Android app theme directory
+# (https://github.com/openedx/openedx-app-android expects 'OpenEdx'). This is addressed in PR https://github.com/openedx/openedx-app-android/pull/335,
+# which is merged into sumac.master and develop. Revert to "PLATFORM_NAME: '{{ PLATFORM_NAME }}'" once the pr changes are merged in master.
```
Comment (DawoudSheraz) [inline tutorandroid/templates/android/build/config/tutor.yaml]: The "master" branch of android app is actually "main". Please update the comment and then squash the commits.

File: tutorandroid/templates/android/build/config/tutor.yaml [hunk 2]
```diff
@@ -68,7 +68,10 @@ BRANCH:
   ALTERNATE_HOST: ''
 
 #Platform names
-PLATFORM_NAME: "{{ PLATFORM_NAME }}"
+# TODO: Temporarily set PLATFORM_NAME based on OPENEDX_COMMON_VERSION to 'OpenEdx' for compatibility with the Android app theme directory
+# (https://github.com/openedx/openedx-app-android expects 'OpenEdx'). This is addressed in PR https://github.com/openedx/openedx-app-android/pull/335,
```
Comment (DawoudSheraz) [inline tutorandroid/templates/android/build/config/tutor.yaml:72]: > (https://github.com/openedx/openedx-app-android expects 'OpenEdx')

Do you have a link that this is the value it expects? The actual issue is that platform name has spaces which cause issues with themeDirs. Therefore, the name is hardcoded to a value with no spaces.

Comment (Abdul-Muqadim-Arbisoft) [reply tutorandroid/templates/android/build/config/tutor.yaml:72]: This [PR](https://github.com/openedx/openedx-app-android/pull/335)  basically ensures theme works properly  without being dependent on the PLATFORM_NAME value, which previously caused issues with theme directories.

**Comment by Abdul-Muqadim-Arbisoft** [comment]
Closing this PR as the changes from [PR #335](https://github.com/openedx/openedx-app-android/pull/335) have already been merged into the main branch. The issue with the 'FIXED' platform name has been resolved with this update.

**Pull request closed without merging**